src/version.js

```diff
@@ -29,6 +30,7 @@ const version = {
       versionApi.getVersion((data) => {
         // Update transaction weight constants
         transaction.updateTransactionWeightConstants(data.min_tx_weight, data.min_tx_weight_coefficient, data.min_tx_weight_k);
+        wallet.updateRewardLockConstant(data.reward_spend_min_blocks);
```

Comment:
We should update the minimum API version.

Reply:
Yes. I usually update this MIN_API_VERSION after merging the full node MR's and updating the VERSION field there.